docker-compose.prod.yml

```diff
@@ -49,30 +49,29 @@ services:
     container_name: opencr
     image: intrahealth/opencr
     ports:
-            #- "8000:3000"
-      - "8000:3000"
+      - "3000:3000"
     depends_on:
       - hapi-fhir
       - es
     environment:
       - NODE_ENV=production
       - HAPI_FHIR_URL=http://hapi-fhir:8080/fhir/metadata
     volumes:
-      - ./opencr/config_production.json:/src/server/config/config_production.json
+      - $docker_containers_shared_dir/conf/opencr/config_production.json:/src/server/config/config_production.json
+      - $docker_containers_shared_dir/conf/opencr/admin.OCRUser.json:/src/resources/ResourcesData/admin.OCRUser.json
```

Comment:
Why are you adding this file?

Reply:
The admin.OCRUser.json? I want to change the default admin user